server/config/test.json

```diff
@@ -1,2 +1,5 @@
 {
+  "host": {
+    "api": "https://opencollective-test-api.herokuapp.com"
```

Comment:
It's still used by client tests. https://github.com/OpenCollective/api/pull/262 takes care of running e2e locally within circleci `api` builds, but we still need to do something similar within circleci `website` and `app` builds. So test server is still needed for a while.